targets/PythonSdk/source/setup.py

```diff
@@ -0,0 +1,20 @@
+import setuptools
+with open("README.md", "r") as fh:
+        long_description = fh.read()
+        
+setuptools.setup(
+    name="playfab",
+    version="0.0.1",
+    author="Todd Bello",
+    author_email="toddbell@microsoft.com",
+    description="PlayFab SDK for Python",
+    long_description=long_description,
```

Comment:
Should probably set this.
For reference, this is what we put in Corona marketplace:
https://marketplace.coronalabs.com/corona-plugins/playfab-combo